### DISCUSSION TITLE
[Stream Deck+] Custom background is covered by HASS entity

### DISCUSSION THREAD

**Discussion by svleeuwen** [cgiesche/streamdeck-homeassistant#206] [Feature Requests]
![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/1196086/46c589e5-4f80-4137-a454-e8d7526399a2)

Maybe there's a way to use a transparent background?

**Comment by cgiesche** [comment]
![grafik](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/09a33e13-dbf2-40e9-a093-6811b1948934)

Easy fix :)

But may look ugly depending on the brightness of the background image:
![grafik](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/6fff0b83-6a4b-4aed-9e8f-78ea8ad449e1)

**Comment by svleeuwen** [comment]
The second option doesn't look that great indeed. But with darker backgrounds it's a lot better than a black bar.

Maybe related: setting a custom background in the icon config doesn't seem to work either. This could allow for a bit more control. We can then add our own border or something.

**Comment by zpriddy** [comment]
I was running into this issue too - I was trying to poke at the code to make a temp fix but I wasnt able to figure out where to make that change. I also couldnt get the custom titles to work on the dial stacks.. but thats probably a different issue or me just not doing it right.. 

Is there any chance you can point me to the change to make to get this working until the fix for the background is pushed?
